docs/installer-stand-alone.md

```diff
@@ -76,6 +76,24 @@ podman run -dt --pod assisted-installer \
 **NOTE**
 * `onprem-environment` is the file downloaded and modified previously
 
+## Start the Image Service
+
+Use podman to run the image service.
+
+```
+podman run -dt --pod assisted-installer \
+  --name image-service \
+  --env-file onprem-environment \
+  --pull always \
+  --restart always \
+  quay.io/edge-infrastructure/assisted-image-service:latest
+```
+
+**NOTE**
+* `onprem-environment` is the file downloaded and modified previously
+* Images will always be downloaded from `IMAGE_SERVICE_BASE_URL` as specified in `onprem-environment`
+* Image download URLs should be retrieved from the infra-env `download_url` field
```

Comment:
can u give an example here maybe?

Reply:
I don't think we need anything else here now that we have the redirect in place.

Reply:
Actually this part can be deleted because @carbonin will fix the API and download api will work as usual.
After Nir will fix the documentation he will add all the relevant parts for v2 anywy.